services/qs-spark-service/src/main/resources/ai/quantumics/ml/DQ_Row_Count_Template.py

```diff
@@ -9,15 +9,13 @@
 
 bucket1 = $SOURCE_BUCKET
 filepath1 = $SOURCE_PATH
-
 bucket2 = $TARGET_BUCKET
 filepath2 = $TARGET_PATH
-
-job_id = $JOB_ID
-rule_id = $RULE_ID
 rule_type_name = $RULE_TYPE_NAME
 level_name = $LEVEL_NAME
 s3Path= $S3_OUTPUT_PATH
+input_acceptance_percentage = $ACCEPTANCE_PER
+input_acceptance_percentage_float = float(input_acceptance_percentage)
 
 file_paths = [
     (filepath1, bucket1),
```

Comment:
if it is only source, then filepath2, bucket2 will be null . should not be added in file_paths array